packages/toolpad-core/package.json

```diff
@@ -18,7 +18,15 @@
   "keywords": [
     "react",
     "mui",
-    "toolpad"
+    "toolpad",
+    "internal tools",
+    "typescript",
+    "crud",
+    "self hosted",
+    "admin panel",
+    "admin framework",
+    "open core",
```

Comment:
I'd limit keywords to describe the intrinsic qualities of the library. Nobody is looking for "open core" software on npm

```suggestion
```